### PR TITLE
Added minus words functionality

### PR DESCRIPTION
**Pull request by bohrium272**

### CODE REVIEW

**Comment by blackforestboi** [comment]
Sweet!

What I just saw would be helpful now is to also remove any functions and code related to the old storage system.

**Comment by bohrium272** [comment]
Will do and add it here

File: src/manifest.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
             "matches": [
                 "<all_urls>"
             ],
-            "js": ["js/compatibility.js","js/pdf.js","js/pdf.worker.js","js/pdftotext.js","js/bundle.js", "js/blacklist2.js","js/background.js","js/lib/stopwords_res.js", "js/lib/readability.js", "js/relevanttext.js", "js/content.js"]
```
Comment (blackforestboi) [inline src/manifest.json:12]: why did you remove bundle.js from the manifest?

Comment (bohrium272) [reply src/manifest.json:12]: Because it is no longer there and every time I would try to install the extension, chrome would give an error.

**Comment by blackforestboi** [comment]
Great Arpit :) Nice, so now we can start our transformation into the beautiful (code)swan :)

A few things that might also be possible to be removed:
 - [make_preloaded() ](https://github.com/arpitgogia/Research-Engine/blob/2e031ae8545d1fb1a9eb24f70998b3e9500bc32c/src/js/background.js#L114)
 - [assert()](https://github.com/arpitgogia/Research-Engine/blob/2e031ae8545d1fb1a9eb24f70998b3e9500bc32c/src/js/background.js#L132)
- [clearCache](https://github.com/arpitgogia/Research-Engine/blob/2e031ae8545d1fb1a9eb24f70998b3e9500bc32c/src/js/background.js#L314)
- inside init ()
    - [get(index)](https://github.com/arpitgogia/Research-Engine/blob/2e031ae8545d1fb1a9eb24f70998b3e9500bc32c/src/js/background.js#L91) (function has no title)
   - window.preloaded
   - window.cache
- [arrax.max](https://github.com/arpitgogia/Research-Engine/blob/2e031ae8545d1fb1a9eb24f70998b3e9500bc32c/src/js/background.js#L16)
- [GT_OBJ](https://github.com/arpitgogia/Research-Engine/blob/2e031ae8545d1fb1a9eb24f70998b3e9500bc32c/src/js/background.js#L12)

File: src/manifest.json (hunk 1 above)
Comment (blackforestboi) [reply src/manifest.json:12]: what do you mean its no longer there? 
Do you build the extension before installing it in the browser?

Comment (bohrium272) [reply src/manifest.json:12]: Bundle.js isn't a part of the extension. That's the case since the last few times I have forked it. When we say "Load extension" to Chrome, it verifies the files mentioned in manifest.json

Comment (blackforestboi) [reply src/manifest.json:12]: yes, thats true but only if you dont build the extension. Bundle.js is important for the history. 

You can see the installation procedure here: https://github.com/WorldBrain/Research-Engine/blob/master/README.md#as-a-developer

Comment (bohrium272) [reply src/manifest.json:12]: Oh OK. I wasn't aware of that. I shall add a new commit.

Comment (blackforestboi) [reply src/manifest.json:12]: Sorry, miscommunication on my part!
